spec/factories/reservations.rb

```diff
@@ -6,6 +6,19 @@
     trait :yesterday do
       reserve_start_at { Time.zone.parse("#{Date.today} 10:00:00") - 1.day }
     end
+
+    trait :later_yesterday do
+      reserve_start_at { Time.zone.parse("#{Date.today} 10:00:00") - 1.day + 1.hour }
+    end
+
+    trait :running do
+      after(:create) do |reservation|
```

Comment:
Why do we need this in an `after(:create)` block? Can we not just set these as normal factory methods?

```
reserve_start_at { 30.minutes.ago }
reserve_end_at { 30.minutes.from_now }
actual_start_at { 30.minutes.ago }
```